src/components/cards/MiniCourseCard.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import {ParsedCourseOverview} from "../../types/Course";
+import {firstLectureOfSubcourse} from "../../utils/CourseUtil";
+import moment from "moment";
+import classes from "./MiniCourseCard.module.scss";
+import Icons from "../../assets/icons";
+import { useHistory } from "react-router-dom";
+
+export const MiniCourseCard = (course: ParsedCourseOverview) => {
+  const firstLecture = firstLectureOfSubcourse(course.subcourse);
+  const firstLectureDate = moment.unix(firstLecture.start).format("DD.MM.");
+
+  const history = useHistory();
+
+  return (
+    <div className={classes.courseCard} onClick={() => history.push(`/courses/${course.id}`)}>
```

Comment:
We should also use that quick and dirty public course detail page (because it is better than nothing). See the following on _master_ branch:
https://github.com/corona-school/web-user-app/blob/2011bc41c94c6670c55848fd63b7b128d91eb0e8/src/routes/PublicCourse.tsx#L202

Reply:
Done.